content/doc/book/blueocean/getting-started.adoc

```diff
@@ -34,15 +34,14 @@ restart**
 image::blueocean/intro/plugin-install.png[Installing Blue Ocean, role=center]
 
 For in-depth description on how to install and manage plugins,
-refer to the <<managing/plugins#, Managing Plugins>> section.
+refer to the <<../managing/plugins#, Managing Plugins>> section.
```

Comment:
This will now go to `../../managing/plugins/`, which is to say `/doc/book/managing/plugins/`.